src/App.js

```diff
@@ -0,0 +1,64 @@
+import SignUp from "components/Auth/SignUp";
```

Comment:
Order of import statements should be
- Libs: react, react-router-dom
- other absolute imports
- other relative imports